From 8f8204b21edbecec159af2a97e247da9326ae9b7 Mon Sep 17 00:00:00 2001
From: Andrey Zhizhikin <andrey.z@gmail.com>
Date: Sat, 9 May 2020 19:56:51 +0000
Subject: [PATCH] opencv: allow compilation against 4.3.x

Backport upstream commit 4cf362e2df0fb809ea0f21dd4a6fbb8b46ca54ef to NXP
fork of gstreamer1.0-plugins-bad.

Original commit link:
https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/commit/4cf362e2df0fb809ea0f21dd4a6fbb8b46ca54ef

Upstream-Status: Backport [4cf362e2df0fb809ea0f21dd4a6fbb8b46ca54ef]

Signed-off-by: Andrey Zhizhikin <andrey.z@gmail.com>
---
 ext/opencv/meson.build | 2 +-
 1 file changed, 1insertions(+), 1 deletions(-)

diff --git a/ext/opencv/meson.build b/ext/opencv/meson.build
index f38b55dfe..6a7836094 100644
--- a/ext/opencv/meson.build
+++ b/ext/opencv/meson.build
@@ -65,7 +65,7 @@ if opencv_found
     endif
   endforeach
 else
-  opencv_dep = dependency('opencv4', version : ['>= 4.0.0', '< 4.2.0'], required : false)
+  opencv_dep = dependency('opencv4', version : ['>= 4.0.0', '< 4.4.0'], required : false)
   opencv_found = opencv_dep.found()
   if opencv_found
     foreach h : libopencv4_headers
-- 
2.20.1

